src/app/screens/__tests__/Home.test.tsx

```diff
@@ -33,7 +33,7 @@ const mockStore = createMockStore({
     history: [historyTx, historyTx]
   },
   persistence: {
-    isPersisted: true
+    rehydratedKeys: []
```

Comment:
How come this can be empty? Shouldn't it require all keys to be there to be an equivalent state to the old `isPersisted=true`

Reply:
The mock store doesn't have any persisted keys